plugins/filter_kubernetes/kube_meta.c

```diff
@@ -290,6 +290,84 @@ static void cb_results(unsigned char *name, unsigned char *value,
     return;
 }
 
+#define CONTAINER_STATUSES "containerStatuses"
+static void extract_container_hash(struct flb_kube_meta *meta,
+                                   msgpack_object status)
+{
+    int i;
+    msgpack_object k, v;
+    int docker_id_len = 0;
+    int container_hash_len = 0;
+    const char *container_hash;
+    const char *docker_id;
+    int name_found = FLB_FALSE;
+    /* Process status/containerStatus map for docker_id, container_hash */
```

Comment:
please document here how the payload looks like to have an easy understanding of the lookup code.

Reply:
done